next/frontend/types/formFileUploadTypes.ts

```diff
@@ -41,10 +58,16 @@ export type FormFileUploadServerFileStatus =
 export type FormFileUploadFileStatus =
   | FormFileUploadClientFileStatus
   | FormFileUploadServerFileStatus
+  | FormFileUploadResponseFileStatus
 
 export type FormFileUploadFileInfo = {
   status: FormFileUploadFileStatus
   fileName: string
   canDownload: boolean
   fileSize: number | null
 }
+
+export enum UploadErrors {
```

Comment:
```suggestion
// values must match translation keys in account.json -> Upload.errors
export enum UploadErrors {
```

please correct the path if I got it wrong